scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -1268,7 +1272,7 @@ function AIDriveStrategyUnloadCombine:startPathfinding(
             -- the same reference everywhere
             local goal = PathfinderUtil.getWaypointAsState3D(target, -xOffset or 0, zOffset or 0)
             self.pathfinder, done, path, goalNodeInvalid = PathfinderUtil.startPathfindingFromVehicleToGoal(
-                    self.vehicle, goal, allowReverse or false, fieldNum, vehiclesToIgnore, {},
+                    self.vehicle, goal, false, fieldNum, vehiclesToIgnore, {},
```

Comment:
Instead of just removing this forgotten variable, we should use getAllowReversePathfinding() (useful for example for trucks)